lib/clAdapters/google-mail/index.js

```diff
@@ -143,14 +143,46 @@ var getAccessToken = function(clientId, adminEmail, userEmail, privateKey) {
   });
 };
 
+var getMoreEmails = function(messages, userEmail, token, apiVersion, additionalFields, emailRequestOptions, firstUri, nextPageToken) {
```

Comment:
It would be great down the road if we could move this adapter into a more object oriented structure, as it would help reduce the need for huge argument lists like this if common data was simply stored on the state.